ballerina-shell/modules/shell-core/src/main/java/io/ballerina/shell/invoker/classload/ClassLoadInvoker.java

```diff
@@ -452,12 +456,14 @@ private ClassLoadContext createVariablesExecutionContext(
      * @param qualifiersAndMetadata Variable snippet qualifiers.
      * @param definedVariables      Variables that were defined.
      * @param globalVarSymbols      All global variable symbols.
+     * @param isAssignedWithVar     type of the variables.
```

Comment:
Need to update the doc comment as well

Reply:
Updated in d3e7d2d557259712428f4f178b8d6bfd67325435